includes/wordpress/class-wp-script-modules.php

```diff
@@ -0,0 +1,359 @@
+<?php
```

Comment:
This class was only introduced in WordPress 6.5. I have loaded it conditionally for users with older versions.

Reply:
An alternative would be to enqueue the JS files as normal scripts, and to use a filter to add the type="module" reference. Possibly this implementation is a bit more straight forward.

See https://stackoverflow.com/questions/73967285/how-do-i-make-my-script-type-module-with-wordpress

Wdyt?